PROJETOS/20251105-github-actions-workflow-modifier/real-autorelease.yaml

```
@@ -0,0 +1,129 @@
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)